package.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "mongoose-delete",
-  "version": "1.0.2",
+  "version": "1.1.0-canary-1",
   "description": "Mongoose soft delete plugin",
-  "author": "Sanel Deljkic <dsanel@gmail.com> (http://dsanel.github.io/)",
+  "author": "Tahera Shorna",
```

Comment:
🤔 I don't think you're supposed to overwrite the author tag on a project. You're certainly a contributor but I'd say that the author tag should stay set to whoever created the project.